src/components/button/button.tsx

```diff
@@ -77,7 +77,7 @@ export class Button
   @Prop() readonly size: "large" | "normal" | "small" = "normal";
 
   @Listen("click", { capture: true })
-  private handleClick(event: UIEvent) {
+  handleClick(event: UIEvent) {
```

Comment:
Can we keep the method private?

Reply:
it isn't private... it's a handle for the event assigned ("clicked").
Can it be the handler and be private at same time?

Reply:
Ok, I understood the issue here. The `@Listen` decorator hides the fact that the `handleClick` method is actually used, and the TS compiler believes that it's not called, thus the warning.
It's ok the way you coded it 👍